spring-web/src/main/java/org/springframework/web/filter/CompositeFilter.java

```diff
@@ -80,10 +80,7 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
 	 */
 	@Override
 	public void destroy() {
-		for (int i = this.filters.size(); i-- > 0;) {
-			Filter filter = this.filters.get(i);
-			filter.destroy();
-		}
+		this.filters.forEach(Filter::destroy);
```

Comment:
This is a breaking change, since it no longer invokes the filters in reverse order.

So please revert this.

Reply:
Done